pagopa/pn-frontend-e2e
# Pn-10375-10376-10377-10378-10379-10380-10381

## Short description

Implemented tests to check the payments in an not yet payed notification

## List of changes proposed in this pull request

- All tests implemented in a single file
- Added a new step to check the "avviso PagoPa" in the payment container
- Added a new step to check the "modello F24" in the payment container
- Added a new step to check the multidestinatario select correct display
- Added a new step to click on the multidestinatario select's first element
- Added some steps to search a notification by IUN code and click it
   - One of these steps has a Timesleep to avoid an error (stale element)

## How to test

Simply run `mvn clean test -D"cucumber.filter.tags=@checkNotificaNonPagata"` and verify the tests work fine.
